src/components/Calculator.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './calculator.css';
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Kindly solve linters instead of disabling them.

```js 


class Calculator extends React.Component {
  constructor(props) {
    super(props);
    this.state = {};
  }
  render() {
    return (....content)
  }
}

Reply:
Hi @andersonlebon
Thank you very much for your feedback.
I have tried to search for solutions for this error and I couldn't find any except for either using functional-based components or disabling linters.
That is the reason why I chose to use a function-based component for the App component as well.
Can you please advise me with solutions or refer me to a page with more details about solving this issue.
Thank you once again.

Reply:
@andersonlebon Thank you the issue is solved